backend/composer/services/cs_ingestion/cs_ingestion_services.py

```diff
@@ -14,8 +14,8 @@
 logger_service = LoggerService()
 
 
-def ingest_statements(update_upstream=False, update_anatomical_entities=False):
-    statements_list = get_statements_from_neurondm(logger_service_param=logger_service)
+def ingest_statements(update_upstream=False, update_anatomical_entities=False, should_overwrite=True, full_imports=[], label_imports=[]):
```

Comment:
issue: We should not use mutable objects as default arguments.

It's better to do:
```
def ingest_statements(update_upstream=False, update_anatomical_entities=False, should_overwrite=True, full_imports=None, label_imports=None):
    if full_imports is None:
        full_imports = []
    if label_imports is None:
        label_imports = []
```

Here's why it's a problem
````
def append_to_list(value, my_list=[]):
    my_list.append(value)
    return my_list

# First call - works as expected
print(append_to_list(1))  # Output: [1]

# Second call - unexpected behavior
print(append_to_list(2))  # Output: [1, 2]

# Third call - still using the same list!
print(append_to_list(3))  # Output: [1, 2, 3]
````